qa/L0_infer/test.sh

```diff
@@ -173,26 +169,23 @@ for TARGET in cpu gpu; do
     done
 
     if [ "$ENSEMBLES" == "1" ]; then
-      if [[ $BACKENDS == *"custom"* ]]; then
-        for BACKEND in $BACKENDS; do
-          if [ "$BACKEND" != "custom" ] && [ "$BACKEND" != "python" ]; then
-              cp -r ${DATADIR}/qa_ensemble_model_repository/qa_model_repository/*${BACKEND}* \
-                models/.
-          elif [ "$BACKEND" == "custom" ]; then
-            cp -r ${DATADIR}/qa_ensemble_model_repository/qa_model_repository/nop_* \
+      for BACKEND in $BACKENDS; do
+        if [ "$BACKEND" != "python" ]; then
+            cp -r ${DATADIR}/qa_ensemble_model_repository/qa_model_repository/*${BACKEND}* \
               models/.
-          fi
-        done
+        fi
+      done
 
-        create_nop_version_dir `pwd`/models
-      fi
+      # Copy identity backend models 
+      cp -r ${DATADIR}/qa_ensemble_model_repository/qa_model_repository/nop_* \
+        models/.
+
+      create_nop_version_dir `pwd`/models
 
       if [[ $BACKENDS == *"graphdef"* ]]; then
         ENSEMBLE_MODELS="wrong_label_int32_float32_float32 label_override_int32_float32_float32 mix_type_int32_float32_float32"
 
-        if [[ $BACKENDS == *"custom"* ]]; then
-          ENSEMBLE_MODELS="${ENSEMBLE_MODELS} batch_to_nobatch_float32_float32_float32 batch_to_nobatch_nobatch_float32_float32_float32 nobatch_to_batch_float32_float32_float32 nobatch_to_batch_nobatch_float32_float32_float32 mix_nobatch_batch_float32_float32_float32"
-        fi
+        ENSEMBLE_MODELS="${ENSEMBLE_MODELS} batch_to_nobatch_float32_float32_float32 batch_to_nobatch_nobatch_float32_float32_float32 nobatch_to_batch_float32_float32_float32 nobatch_to_batch_nobatch_float32_float32_float32 mix_nobatch_batch_float32_float32_float32"
```

Comment:
@CoderHam I think now you use savedmodel for the ensemble? Just change the conditional to check if "savedmodel" is in BACKENDS here and elsewhere. We don't want to run "savedmodel" if the script is invoked without specifying it.

Reply:
These ensembles now use the identity backend model instead of the legacy custom backend addsub